x-pack/plugins/synthetics/common/rules/alert_actions.ts

```diff
@@ -37,51 +36,46 @@ const { MONITOR_STATUS } = ACTION_GROUP_DEFINITIONS;
 
 export type RuleAction = Omit<RuleActionOrig, 'actionTypeId'>;
 
-const getRecoveryMessage = (selectedMonitor: Ping) => {
-  return i18n.translate('xpack.synthetics.alerts.monitorStatus.recoveryMessage', {
-    defaultMessage: 'Monitor {monitor} with url {url} has recovered with status Up',
-    values: {
-      monitor: selectedMonitor?.monitor?.name || selectedMonitor?.monitor?.id,
-      url: selectedMonitor?.url?.full,
-    },
-  });
-};
```

Comment:
instead of hard coding recovery message, now the recovery message will be fetched from context.